centrallix/osdrivers/objdrv_xml.c

```diff
@@ -214,7 +214,7 @@ typedef struct
 /*** GLOBALS ***/
 struct
     {
-    XHashTable	cache;
+    XHashQueue	cache;
```

Comment:
Should also #include "cxlib/xhashqueue.h" with the other includes at the beginning of the file, mainly for good hygiene.  The file is already included from obj.h, which is why things are working.

Reply:
Noted